inhouse-init.sql

```diff
@@ -97,7 +97,8 @@ CREATE TABLE "public"."players" (
     "name" character varying(50) NOT NULL,
     "win" integer NOT NULL,
     "loss" integer NOT NULL,
-    "sp" integer NOT NULL
+    "sp" integer NOT NULL,
+    "show_rank" boolean
```

Comment:
can we set it to default to false as well?

I believe syntax is 
```sql
ALTER TABLE public.players ALTER COLUMN show_rank SET DEFAULT FALSE;
```

and it can go down with the alter table stuff on 141-147